package.json

```diff
@@ -9,11 +9,11 @@
   "icon": "images/csharpIcon.png",
   "preview": false,
   "bugs": {
-    "url": "https://github.com/microsoft/vscode-dotnettools/issues"
+    "url": "https://github.com/dotnet/vscode-csharp"
   },
   "repository": {
     "type": "git",
-    "url": "https://devdiv@dev.azure.com/devdiv/DevDiv/_git/vscode-csharp-next"
+    "url": "https://github.com/dotnet/vscode-csharp"
```

Comment:
If this is supposed to be a clone URL, github puts a .git on the end. I think the regular URL works for clones but I'm not sure if that's something they don't want people doing.

Reply:
Nope - this should just be a link to the repo in the marketplace page.